guidesmiths/dangerjs-wrapper
# chore/update-typescript-configuration

### What’s the focus of this PR

- Update the used packages.
- Update the TypeScript configuration.
- Update the `package.json` configuration in order to generate a valid NPM module.

### How to review this PR

- Review the content of the `CHANGELOG.md` file in order to figure out which have been the applied modifications.
- Download the branch and build the project.
- Run Verdaccio locally and deploy the package on it.
  - `"npm:local-unpublish": "npm unpublish dangerjs-wrapper --force --registry http://localhost:4873"`
  - `"npm:local-publish": "npm run build && npm run npm:local-unpublish && npm publish --registry http://localhost:4873"`
- Go to the web project.
- Edit the `.npmrc` file in order to keep active only the option `@ecommerce:registry=http://localhost:4873/`
- Login into Verdaccio: `npm login`.
- Install the package in a specific testing branch.
- Verify the package is installed successfully and its exposed function are available and correctly typed.

### Related work items

- No defined.

### Before submitting this PR, I made sure:

- [X] I'm using our guidelines
- [ ] I've added tests
